go-apps/meep-rnis/server/rnis.go

```diff
@@ -1393,23 +1453,24 @@ func rabEstSubscriptionsDELETE(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusNoContent)
 }
 
-func rabRelSubscriptionsGET(w http.ResponseWriter, r *http.Request) {
+/*
+func cellChangeSubscriptionsGET(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
 	vars := mux.Vars(r)
 	subIdParamStr := vars["subscriptionId"]
 
-	var response InlineResponse2009
-	var rabRelSubscription RabRelSubscription
-	response.RabRelSubscription = &rabRelSubscription
+	var response InlineResponse2004
```

Comment:
This type should no longer exist.

Reply:
was one left part of commented code... but i removed it now.. so none appear in latest build